Mathlib/Algebra/Category/Ring/Center.lean

```diff
@@ -0,0 +1,135 @@
+/-
+Copyright (c) 2025 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang
+-/
+import Mathlib.CategoryTheory.Adjunction.Limits
+import Mathlib.Algebra.Category.ModuleCat.Limits
+
+/-!
+# A categorical description of the center of a ring
+
+In this file we prove that the center of a ring `R` is isomorphic to `End (𝟭 R-Mod)` the
+endomorphism ring of the identity functor on the category of `R`-modules. Consequently, the ring
+structure of a commutative ring is complete determined by its module category.
+
+## Main results
+
+- `Subring.centerEquivEndIdFunctor`: the center of a ring `R` is isomorphic to `End (𝟭 R-Mod)`.
+- `RingEquiv.ofModuleCatEquiv`: if two commutative rings have equivalent module categories, they are
+  isomorphic as rings.
+
+-/
+
+universe u u' v v'
+
+variable (R : Type u) [Ring R]
+
+open CategoryTheory
+
+/--
+For any ring `R`, the center of `R` is isomorphic to `End (𝟭 (ModuleCat R))`, the endomorphism ring
+of the identity functor on the category of `R`-modules.
+-/
+@[simps]
+def Subring.centerEquivEndIdFunctor : Subring.center R ≃+* End (𝟭 (ModuleCat.{max v u} R)) where
```

Comment:
Can you use `Small` rather than `max v u`? Same below.

Reply:
done, thanks